docs/framework/data/adonet/ef/language-reference/from-entity-sql.md

```diff
@@ -131,7 +131,7 @@ from (C as c join D as d) cross apply c.Names as e
 from (C as c join D as d) cross apply c.Names as e  
 ```  
   
- In [!INCLUDE[esql](../../../../../../includes/esql-md.md)] (unlike [!INCLUDE[tsql](../../../../../../includes/tsql-md.md)]), the `FROM` clause only introduces the aliases into scope. Any references to columns (properties) of these collections must be qualified with the alias.  
+ In [!INCLUDE[esql](../../../../../../includes/esql-md.md)](unlike [!INCLUDE[tsql](../../../../../../includes/tsql-md.md)]), the `FROM` clause only introduces the aliases into scope. Any references to columns (properties) of these collections must be qualified with the alias.  
```

Comment:
I agree with @pkulikov, @v-alje. This can't be a bulk change operation. Along with removal of valid whitespace characters, these changes introduced six build breaks.  My sense is that the best approach is to close this PR and submit a new PR that doesn't do a bulk change in eliminating whitespace characters.